ui/app/AppLayouts/Chat/popups/community/InviteFriendsToCommunityPopup.qml

```diff
@@ -35,6 +35,9 @@ StatusModal {
         contentItem.contactListSearch.noContactsRect.visible = !contentItem.contactListSearch.existingContacts.visible;
     }
 
+    margins: 32
+    height: 550
```

Comment:
This is very bad approach here, but the reason is `ContactsListAndSearch` component (which is part of `CommunityProfilePopupInviteFriendsPanel` component) which is developed in a very unmaintainable way, and as is, used on multiple places across the app and that was the only reason why I fixed the issue #6106 in this embarasing way. 

We should redo `ContactsListAndSearch` in a more appropriate way.